src/Stateless/Reflection/ActionInfo.cs

```diff
@@ -18,13 +18,16 @@ public class ActionInfo
         internal static ActionInfo Create<TState, TTrigger>(StateMachine<TState, TTrigger>.EntryActionBehavior entryAction)
         {
             StateMachine<TState, TTrigger>.EntryActionBehavior.SyncFrom<TTrigger> syncFrom = entryAction as StateMachine<TState, TTrigger>.EntryActionBehavior.SyncFrom<TTrigger>;
-
             if (syncFrom != null)
                 return new ActionInfo(entryAction.Description, syncFrom.Trigger.ToString());
-            else
-                return new ActionInfo(entryAction.Description, null);
+
+            StateMachine<TState, TTrigger>.EntryActionBehavior.AsyncFrom<TTrigger> asyncFrom = entryAction as StateMachine<TState, TTrigger>.EntryActionBehavior.AsyncFrom<TTrigger>;
+            if (asyncFrom != null)
+                return new ActionInfo(entryAction.Description, asyncFrom.Trigger.ToString());
+
+            return new ActionInfo(entryAction.Description, null);
         }
-        
```

Comment:
Whitespace warning

Reply:
I think Rider removed this automatically?

Reply:
Ahh yes, sorry, I think the coding style in this repo is to have no empty whitespace at the end of a line. It's a nitpick but it would be best practice to remove it.

Reply:
I think I'm being stupid 🤣.
Line 27 used to have some whitespace, but my PR removed it?
Could you point out where I need to make changes, please?

Reply:
As you say, your PR removed the whitespace so I think it's all good. @crozone?

Reply:
Yeah I think it's fine now. LGTM.